var/spack/repos/builtin/packages/edm4hep/package.py

```diff
@@ -69,7 +70,9 @@ class Edm4hep(CMakePackage):
     depends_on("podio@1:", when="@0.99:")
     depends_on("podio@0.15:", when="@:0.10.5")
     for _std in _cxxstd_values:
-        depends_on("podio cxxstd=" + _std, when="cxxstd=" + _std)
+        assert len(_std) == 1
```

Comment:
I don't think the assert is needed here, right? Usually we avoid asserts in packages (and elsewhere for that matter) and handle the underlying failure modes when they can be identified. What's the failure mode here that you're worried about?

Reply:
Ah, this is a leftover from developing / debugging. Removed.